feature/bgp/gracefulrestart/gracefulrestart.go

```diff
@@ -27,8 +27,9 @@ import (
 
 // GracefulRestart struct to store OC attributes.
 type GracefulRestart struct {
-	noc oc.NetworkInstance_Protocol_Bgp_Neighbor_GracefulRestart
-	poc oc.NetworkInstance_Protocol_Bgp_PeerGroup_GracefulRestart
+	noc    oc.NetworkInstance_Protocol_Bgp_Neighbor_GracefulRestart
+	poc    oc.NetworkInstance_Protocol_Bgp_PeerGroup_GracefulRestart
+	global oc.NetworkInstance_Protocol_Bgp_Global_GracefulRestart
```

Comment:
maybe name this goc to match the other field names.

Reply:
Changed it to "goc". Done!